azure-pipelines.yml

```diff
@@ -92,6 +92,13 @@ jobs:
           MB_ML_VER: "2014"
           NP_BUILD_DEP: "numpy==1.19.3"
           NIGHTLY_BUILD: "true"
+        py_3.10_64:
```

Comment:
are we doing 32bit wheels for py3.10?

Reply:
Numpy isn't right now. We'll wait for them.